tests/kubernetes-base-addons/addons/traefik/1.7.x/traefik-spdy/exec-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+cleanup() {
+    rc=$?
+    rm ${TMPKUBECONFIG}
+    exit $rc
+}
+
+OUTDIR=$(ARTIFACT:-/tmp)
+SERVER=$(kubectl -n kubeaddons get svc traefik-kubeaddons \
+    -o jsonpath="{.status.loadBalancer.ingress[0].ip}{.status.loadBalancer.ingress[0].hostname}")
+SECRET=$(kubectl -n "${NAMESPACE}" get serviceaccount kuttlaccount -o jsonpath="{ .secrets[0].name }")
+TOKEN=$(kubectl -n "${NAMESPACE}" get secret "${SECRET}" -o go-template="{{.data.token | base64decode }}")
+TMPKUBECONFIG=$(mktemp)
```

Comment:
not even a nit, but his could be done with `kubectl -s ${SERVER} --token=${TOKEN}`

Reply:
This wasn't working when I tested it. It was using the wrong credentials.